src/lxc/lxccontainer.h

```diff
@@ -1017,6 +1017,22 @@ int list_active_containers(const char *lxcpath, char ***names, struct lxc_contai
  */
 int list_all_containers(const char *lxcpath, char ***names, struct lxc_container ***cret);
 
+struct lxc_log_conf {
+	const char *name;
+	char *file;
+	char *priority;
+	const char *prefix;
+	int quiet;
+	const char *lxcpath;
+};
+
+/*!
+ *\brief Initialize the log
+ *
+ *\param log lxc log configuration.
+ */
+int lxc_log_init(struct lxc_log_conf log);
```

Comment:
We will not change anything that `log` holds. If you prefer pointer, I will change it later today, thanks